presto-main/src/main/java/io/prestosql/operator/HttpPageBufferClient.java

```diff
@@ -322,8 +322,9 @@ public void onSuccess(PagesResponse result)
                         }
 
                         if (!isNullOrEmpty(taskInstanceId) && !result.getTaskInstanceId().equals(taskInstanceId)) {
-                            // TODO: update error message
-                            throw new PrestoException(REMOTE_TASK_MISMATCH, format("%s (%s)", REMOTE_TASK_MISMATCH_ERROR, fromUri(uri)));
+                            String msg = format("%s (%s). Expected TaskInstanceId: %s, received TaskInstanceId: %s", REMOTE_TASK_MISMATCH_ERROR, fromUri(uri), taskInstanceId, result.getTaskInstanceId());
```

Comment:
Thanks for updating. Please inline this (there is no benefit from this being a variable).
(this is how we typically format these -- https://github.com/prestosql/presto/blob/10791d0a9d6f2aabbcfc9e1c0c54cf18adb11b76/presto-hive/src/main/java/io/prestosql/plugin/hive/HivePartitionManager.java#L202-L205)